common/colors.css

```diff
@@ -10,6 +10,10 @@
     --bg-color: #fcfcfc;
     --bg-secondary : #ffffff;
 
+    --red : #f87171;
+    --green : #34d399;
+    --yellow : #facc15;
+
     /* --primary-color : #03A9F4;
     --primary-dark : #B3E5FC;
     --primary-light : #0288D1;
```

Comment:
I suggest naming your variables as `--primary-color` or `--secondary-color` like you did in your commented code. Also, you can delete the commented code if it is not required.

Reply:
Thanks @vidhi-thakur I'll update them with better names.
and I'll be using the commented colors for dark mode later.